getting_started_scenarios/ssm_scenario/Readme.md

```diff
@@ -0,0 +1,42 @@
+# AWS Systems Management (SSM) Getting Started Scenario
+
+## Introduction
+This Java V2 AWS Systems Manager (SSM) getting started scenario demonstrates how to interact with the AWS Systems Manager service using the AWS SDK for Java (v2). The scenario covers various operations such as creating a maintenance window, creating an SSM document, sending a command to a managed node, creating an OpsItem, updating an OpsItem, and deleting SSM resources.
```

Comment:
Should this be language agnostic?
You describe it as Java specific. Won't this be confusing when other languages are added?